### PR TITLE
Eventtrack rename

### PR DESCRIPTION
**Pull request by reichlfl**
Rename EventTrack, TracepointTrack and ThreadStateTrack.
This also fixes a type in the E2E test for ThreadState that lead to a failure.

### CODE REVIEW

**Review by danielfenner — APPROVED** [review]
lgtm

File: src/OrbitGl/CMakeLists.txt [hunk 1]
```diff
@@ -28,7 +28,7 @@ target_sources(
          DataViewTypes.h
          Disassembler.h
          DisassemblyReport.h
-         EventTrack.h
+         CallstackThreadBar.h
```
Comment (dpallotti) [inline src/OrbitGl/CMakeLists.txt:31]: Move up to keep alphabetical order.

File: src/OrbitGl/CMakeLists.txt [hunk 2]
```diff
@@ -89,7 +89,7 @@ target_sources(
           DataView.cpp
           Disassembler.cpp
           DisassemblyReport.cpp
-          EventTrack.cpp
+          CallstackThreadBar.cpp
```
Comment (dpallotti) [inline src/OrbitGl/CMakeLists.txt:92]: Move up for alphabetical order.

File: src/OrbitGl/ThreadTrack.h [hunk 3]
```diff
@@ -66,9 +66,9 @@ class ThreadTrack final : public TimerTrack {
   void UpdatePositionOfSubtracks();
   void UpdateMinMaxTimestamps();
 
-  std::shared_ptr<orbit_gl::ThreadStateTrack> thread_state_track_;
-  std::shared_ptr<orbit_gl::EventTrack> event_track_;
-  std::shared_ptr<orbit_gl::TracepointTrack> tracepoint_track_;
+  std::shared_ptr<orbit_gl::ThreadStateBar> thread_state_track_;
+  std::shared_ptr<orbit_gl::CallstackThreadBar> event_track_;
+  std::shared_ptr<orbit_gl::TracepointThreadBar> tracepoint_track_;
```
Comment (dpallotti) [inline src/OrbitGl/ThreadTrack.h:71]: I guess that these should be renamed.

**Review by dpallotti — APPROVED** [review]
Thanks! I think you forgot to rename the fields of `ThreadTrack`. Looks good otherwise.

File: src/OrbitGl/ThreadTrack.h (hunk 3 above)
Comment (reichlfl) [reply src/OrbitGl/ThreadTrack.h]: Good catch, I missed those... changed, thanks!

**Pull request merged**